be/src/formats/parquet/group_reader.cpp

```diff
@@ -48,15 +56,51 @@ Status GroupReader::init() {
     return Status::OK();
 }
 
+bool GroupReader::_can_use_any_column() const {
+    return _param.can_use_any_column && (_need_skip_rowids == nullptr || _need_skip_rowids->empty());
+}
+
+void GroupReader::_init_use_any_column() {
+    VLOG_FILE << "[xxx] init_use_any_column = OK";
+    UseAnyColumnContext& ctx = _use_any_column_ctx;
+    ctx.row_count = _row_group_metadata->num_rows;
+
+    if (_param.read_cols.size() != 0) {
+        DCHECK_EQ(_param.read_cols.size(), 1);
+        const auto& c = _param.read_cols[0];
+        const auto& slots = _param.tuple_desc->slots();
+        ctx.slot_desc = slots[c.col_idx_in_chunk];
+    }
+}
+
+Status GroupReader::_build_chunk_on_use_any_column(ChunkPtr* chunk, size_t* row_count) {
+    UseAnyColumnContext& ctx = _use_any_column_ctx;
+    if (ctx.row_count == 0 || ctx.slot_desc == nullptr) {
+        *row_count = 0;
```

Comment:
I think this situation should not happen, add an extra `DCHECK(false)` here.

Reply:
Each RowGroup must have `num_rows > 0`

Reply:
Or we can check `num_rows > 0` and `slot_desc != nullptr` in `_init_use_any_column()`

Reply:
I forget when it happens, but it indeed happens when I'm doing test.